sound/soc/sof/sof-client.h

```diff
@@ -0,0 +1,64 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
ditto

Reply:
see the structure line 14 below, it's an extension of the Linux-specific auxiliary device. I am not comfortable with dual-license support here, it raises many questions and brings limited benefits.